src/App.tsx

```diff
@@ -1,203 +1,128 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import cn from 'classnames';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { Filter } from './components/Filter';
+import { Filters } from './constants/filter';
+import { TodoList } from './components/TodoList';
+import { NewTodo } from './components/NewTodo';
+
+const getPreparedTodos = (todos: Todo[], filter: Filters) => {
+  const newTodos = [...todos];
+
+  if (filter !== Filters.all) {
```

Comment:
enum values are usually capitalized
```suggestion
  if (filter !== Filters.All) {
```